default/setup.xml

```diff
@@ -5,18 +5,13 @@
     <type>text</type>
   </input>
   <input field="password">
-    <label>Crittercism Password</label>
-    <type>password</type>
-  </input>
-</block>
-<block title="Crittercism API Info" endpoint="configs/conf-crittercism_integration" entity="api">
-  <input field="clientID">
-    <label>This is the Client ID that you received from Crittercism Support</label>
+    <label>Crittercism OAuth Token</label>
     <type>text</type>
   </input>
-<text> Notes:</text> 
-<text> - To change the Credentials or Crittercism API Key later, first remove the entry from SPLUNK_HOME/etc/apps/crittercism/local/app.conf and restart Splunk.
-</text><text> - The script may take up to 10 minutes to start getting data into the Crittercism dashboards.
+<text> Notes:</text>
+    <text> - To generate your Crittercism OAuth Token, please visit https://app.crittercism.com/developers/user-settings and follow the instructions on the page.</text>
+    <text> - To change the Credentials or Crittercism API Key later, first remove the entry from SPLUNK_HOME/etc/apps/crittercism/local/app.conf and restart Splunk.
```

Comment:
Is there a way to allow them to paste a new token in rather than editing that line?